### ISSUE TITLE
Inscription compte inactif

### ISSUE THREAD

**Issue by machine62** [OGSteam/tool-OGSMarket#30]
Activer par défaut le compte après enregistrement ....

Sinon, on decouvre l'outil et on ne peux pas l'utiliser...
apres recherche on découvre que darknoon est administrateur mais pas d 'info sur comment le contacter ...

**Comment by darknoon29** [comment]
On parle ici du market hébergé sur ogspy.fr?

**Comment by machine62** [comment]
OUi, je parle de celui hebergé, je pensais qu'on avait tout centralisé pour cet outil ..

**Comment by darknoon29** [comment]
On peut quasiment dire ça vu que personne ne l'installe à la main 😁
